### DISCUSSION TITLE
Cancel Modrinth presence on X (Twitter)

### DISCUSSION THREAD

**Discussion by brawaru** [modrinth/.github#164] [Feature requests]
⚠️ **CONTENT WARNING FOR MENTIONS OF ANTISEMITISM, RACISM AND OTHER HATEFUL MOVEMENTS**

<details>
<summary>Potentially triggering content describing the current situation on X (Twitter) and call for Modrinth to leave the platform</summary>

<p>Modrinth has been known to have a very diverse user base and team. As a Discord server moderator, I know that we very much value inclusivity and safety of all our members and do not condone any form of discrimination, such as racism, sexism, homophobia, transphobia, antisemitism, etc. And I hope we continue to be the platform where creators and community feel safe and welcome.
<p>​
<p>That being said, Modrinth is currently maintaining a presence on X (formerly known as Twitter). Unfortunately, ever since the takeover by Elon Musk X (Twitter) has seen a huge number of controversial events and dangerous regressions in moderation against hateful content. In fact, Elon Musk himself frequently interacts with various hateful individuals and fulfills their requests.
<p>​
<p>Right as I am writing this, there is antisemitic campaign running on X targeting ADL after Elon Musk engaged with the user who is known to be antisemitic, and expressed his support towards hate against ADL. Users have also noticed an increase in racist and antisemitic ads like ‘protect white Christian values’.
<p>​
<p>Even putting that aside, there have been numerous concerning events, like recent privacy policy update where Twitter now is going to collect data on people's employment, as well as their biometry.
<p>​
<p>I understand that social presence is important, but it strikes me wrong that Modrinth is using X (Twitter), which has turned into a literal nazi bar. I believe you are better than that, I believe you should be one of the companies to leave this site ASAP over dangerous events that are currently happening there. Therefore with this post I call for Modrinth to leave X.
<p>​
<p>Other users can share their opinion here using upvotes, reactions and replies.

</details>

**Comment by MagmaSlime123** [comment]
*long sigh* And here we go again with the drama... Why he bought Twitter is beyond me. All it caused is this, and other sh!t to go down as well.

**Comment by some-du6e** [comment]
fun fact: racism has increased since he bought it

**Reply by MagmaSlime123** [reply]
Oof...

**Reply by brawaru** [reply]
That is very true, and not only racism, all kinds of bigoted and harmful content. Before him Twitter at least would somewhat react to reports, but like not month after his takeover there began huge issues with that. Eventually Twitter stopped reacting to reports at all. I deleted my Twitter accounts when they unbanned nazi accounts and every post featuring any kind of minority got a huge amount racists, homophobes, transphobes, and literal nazis in the replies. 

<details><summary>Potentially triggering content describing unmoderated content on Twitter</summary>
<p>

To give example how bad it got, I reported accounts that called to hang gay people or posted LGBTQ+ flags set on fire or stepped on, that told trans people to off themselves, and all I received are variations of this RIDICULOUS in the context letter:

> Hello,
> 
> After reviewing the available information, we want to let you know [REDACTED] hasn't broken our [safety policies](https://web.archive.org/web/20221124042205/https://help.twitter.com/en/rules-and-policies/twitter-rules). We know this isn't the answer you're looking for. If this account breaks our policies in the future, we'll notify you.
> 
> You can [block](https://web.archive.org/web/20221124053242/https://help.twitter.com/en/using-twitter/blocking-and-unblocking-accounts) the account, which means they won't be able to follow you, see your Tweets or message you.
>  
> Reports like this inform our policies, and we're always reviewing them and how we apply them. We hope you'll continue to make reports if you see things that might break our policies.
>  
> Here's a summary of what isn't allowed on Twitter, according to our safety policies.
>  
>
> - Threatening violence against someone or a group of people
> - Celebrating or praising violence
> - Harassing someone or encouraging people to harass someone
> - Wishing harm on someone
> - Promoting violence, threatening or harassing people because of their identity (like race or gender)
> - Promoting or encouraging suicide or self-harm
> - Images or videos that show sexual violence and assault
> - Child sexual exploitation
> - Threatening or promoting terrorism or violent extremism
> 
> We know we don't always get it right. So if you think we made a mistake, you can report them again.
>  
> Thanks,
> Twitter

</p>

<p>MY BROTHER IN CHRIST, the kind of content I reported is literally in your freaking letter!!</p>

<p>

Is this the case these days? It still very much is:

<a href="https://bsky.app/profile/stevesilberman.bsky.social/post/3k6r2th6vlv25"><img alt="Screenshot of post by Steve Silberman (@stevesilberman.bsky.social):&#10;Dear remaining Twitter/X advertisers: If you're onboard with associating your trusted brand with gassing the Jews, you've come to the right place!&#10;&#10;Attached to that post is an image of litter by Twitter, where they say that gasthejews6969 hasn't broken Twitter safety policies." src="https://i.imgur.com/vCMoWkI.jpeg"></a>

</p>
</details>

**Comment by JadeWhite** [comment]
Modrinth is in its infancy at the moment and needs the exposure to be commercially viable. Even if it was taken off Twitter, Curseforge, Prism Launcher, FTB etc. certainly won't be leaving Twitter. Leaving Twitter would also leave an opening for a non-official (and uncontrolled) Modrinth account to start up in its wake. Don't see how this ends up being anything other than a loss move for Modrinth tbh.

**Reply by osfanbuff63** [reply]
Well, the current official account doesn't have to be deleted, it can just not be used anymore

**Comment by brawaru** [comment]
![Elon Musk on Twitter: ‘Reinstate Alex Jones on this platform? Vox Populi, Vox Dei.’ Attached to his Tweet is a pool with 1.9M votes, 70% of which are Yes.](https://github.com/modrinth/.github/assets/10401817/28733e27-da5c-48b0-b2a3-0e14d0282d32)

![Elon Musk on Twitter: ‘I vehemently disagree with what he said about Sandy Hook, but are we a platform that believes in freedom of speech or are we not? That is what it comes down to in the end. If the people vote him back on, this will be bad for X financially, but principles matter more than money.’](https://github.com/modrinth/.github/assets/10401817/070ee33e-6e5c-4769-9f50-8cec7c5607ce)

At least someone has principles when it comes to Twitter. Oh...

**Reply by brawaru** [reply]
Hey, but it's commercially viable, that's all that matters!
